edit-post/components/meta-boxes/meta-boxes-area/style.scss

```diff
@@ -85,8 +58,8 @@
 		right: 20px;
 		z-index: z-index( '.edit-post-meta-boxes-area .spinner');
 	}
-}
 
-.edit-post-meta-boxes-area__clear {
-	clear: both;
+	&__clear {
```

Comment:
FWIW I found the previous form, with independent selector, to be more readable. I tend to avoid `&` except for applying modifiers (`&.is-empty`, `&:disabled`)

Reply:
No strong preferences. I tend to put everything inside a block when there's a prefix to use. Will resolve conflicts and wait.